src/app/modules/reports/components/time-entries-table/time-entries-table.component.spec.ts

```diff
@@ -107,6 +112,14 @@ describe('Reports Page', () => {
       });
     });
 
+    it('The data should be displayed as a multiple of hour when column is equal to 3', () => {
+      expect(component.bodyExportOptions(durationTime, 0, 3, 0)).toMatch(/^\d+\.\d{0,2}$/);
```

Comment:
Could you assign this values: 0, 3, 0 and the regex expression with variables, please? It's difficult to understand what they are about

Reply:
These changes will be in the next commit, thank you @faustocv.